F1WM/Controllers/SeasonController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using F1WM.ApiModel;
+using F1WM.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace F1WM.Controllers
+{
+    [Route("api/[controller]")]
+    public class SeasonController : ControllerBase
```

Comment:
I'd stick to naming scheme from other controllers (endpoints), so that it would be Season**s**Controller (producing api/season**s**/* endpoint addresses). The comment applies all way down to services and repositories layer (it would be nice if you renamed them respectively), but only here has actual impact on functionality.